src/index.js

```diff
@@ -25,7 +25,6 @@ exports.MessageFlags = require('./util/MessageFlags');
 exports.Options = require('./util/Options');
 exports.Permissions = require('./util/Permissions');
 exports.RateLimitError = require('./rest/RateLimitError');
-exports.SnowflakeUtil = require('./util/SnowflakeUtil');
```

Comment:
We don't intend to expose it?

Reply:
What name should we expose it as? `SnowflakeUtil` (same as now)? `DiscordSnowflake` (same as dep)?

Reply:
I don't really care honestly, I was just wondering if this was a conscious decision

Reply:
I was thinking among the lines of letting users install the dependency if they need it, however, we'd naturally expose a part of the library since `TwitterSnowflake` is not really related to discord.js.

However, `@discordjs/collection`'s `Collection` is re-exported, although it might have been for retro-compatibility since we exported it for many versions, the same can't really be said with the new library, which has similar methods but different inputs and outputs.

I don't really care either, I can see the use of re-exporting `DiscordSnowflake` as `SnowflakeUtil` for ease of transition and exposing something that might be useful to users, were they to need it, but at the same time, this is mostly an utility class we use internally, like `AsyncQueue` from `@sapphire/async-queue`, which is not exported.

Reply:
Yeah the problem with them installing deps themselves is that version numbers may diverge and if it's useful for the end user, a re-export solves that problem.

Reply:
Re-exported in b0d1c8baebe147476ae324729bba12383cbaf079 as `SnowflakeUtil`, I figured we didn't need to use `Discord` in the name as this is inside a Discord-focused library.